src/app/components/oeb-button.component.ts

```diff
@@ -6,7 +6,7 @@ import { NgIf } from '@angular/common';
   selector: 'oeb-button',
   standalone: true,
   imports: [HlmButtonDirective, NgIf],
-  template: `<button class="tw-relative" hlmBtn [disabled]="disabled" [size]="size" [variant]="variant"><img *ngIf="img" class="md:tw-h-[30px] tw-h-[20px] tw-pr-4" [src]="img"/><span [innerHTML]="text"></span></button> `,
+  template: `<button class="tw-relative tw-w-max" hlmBtn [disabled]="disabled" [size]="size" [variant]="variant"><img *ngIf="img" class="md:tw-h-[30px] tw-h-[20px] tw-pr-4" [src]="img"/><span [innerHTML]="text"></span></button> `,
```

Comment:
I dont think we should use width max here since it would affect all other buttons. 
There are two options:
- We can allow to overwrite existing classes by passing the class to oeb-button. 
- Add a full-width variant/size to the ButtonComponent

Reply:
I added `width` variant.